zavod/zavod/verifiers/assertions.py

```diff
@@ -0,0 +1,66 @@
+from typing import Dict, Any, Optional, cast
+from zavod.context import Context
+
+from zavod.entity import Entity
+from zavod.meta.assertion import Assertion, Comparison, Metric
+from zavod.exporters.statistics import Statistics
```

Comment:
@pudo is this a bit too cheeky or is it reasonable to leave this like this til we actually want to mess with the numbers used by the assertions and debuild then?